var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -84,6 +88,7 @@ class Qmcpack(CMakePackage, CudaPackage):
     conflicts('^intel-mkl+ilp64',
               msg='QMCPACK does not support MKL 64-bit integer variant')
 
+
```

Comment:
There are a few flake8 fixes that are causing the CI too fail. Here is one:
var/spack/repos/builtin/packages/qmcpack/package.py:92: [E303] too many blank lines (2)

Reply:
will fix in my next push